test/utils/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3"
+
+services:
+
+  geth_node_origin:
+    image: augurproject/dev-node-geth:v1.8.18
+    ports:
+      - "8546:8545"
```

Comment:
Does this config mean 2 nodes of geth will be running on ports 8545 and 8546?
Can we use latest geth release version v1.8.21 ? 
 https://github.com/ethereum/go-ethereum/releases

I think travis also has docker support. Just curious any specific reason not going that route?
https://docs.travis-ci.com/user/docker/

Reply:
Yeah, that's what I was talking about above.

But I confused the git history for inclusion of that (13 days ago) with the age of the latest tag (10 days) and thought the latest existed when mosaic chose to go with v1.8.18 and not the other way around 😳 Apologies!

I will update for using the latest tag. Will also mention this to the mosaic team to see if they see any issues/will do the same.

**Question**: will using the `latest` tag, rather than an exact tag, risk issues, as noted above in your comment about wait-port?

Reply:
``
Question: will using the latest tag, rather than an exact tag, risk issues, as noted above in your comment about wait-port?
``
I am not sure on that :(

Reply:
Hmm... then in the absence of certainty on this point, I would vote we stick with v1.8.18, because that is consistent with mosaic and it is working.

Additionally, geth appears to think not upgrading might be a sensible choice (from the release notes):
> "Geth 1.8.21 is an emergency hotfix release ... The reason is a last minute reentrancy vulnerability ... If you don't feel comfortable with upgrading to Geth 1.8.21 ..."

Reply:
Agree. Let's keep v1.8.18 👍